NewSpring/ApollosProject
# (DEV-3598) Adds Auto Versioning

## DESCRIPTION

### What does this PR do, or why is it needed?

Adds `yarn bump` command that will auto version `package.json` and native files as well as update the `CHANGELOG.md` automatically.

### How do I test this PR?

Run `yarn bump` just to see what happens (don't commit the changes)

---

> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO


## REVIEW


**Manual QA**


---

> The purpose of PR Review is to _improve the quality of the software._
